src/frontend/src/app/core/ui/tooltip/tooltip.directive.ts

```diff
@@ -45,6 +46,19 @@ export class TooltipDirective implements OnInit, OnChanges {
     }
   }
 
+  @HostListener('click')
+  public showOnMobileScreen(): void {
+    if (
+      this.tooltipText !== null &&
+      window.matchMedia('(pointer: coarse)').matches
+    ) {
+      const tooltipPortal = new ComponentPortal(TooltipComponent);
+      this.tooltipRef = this.overlayRef.attach(tooltipPortal);
+
+      this.tooltipRef.instance.tooltipText = this.tooltipText;
+    }
+  }
+
   @HostListener('mouseleave')
   public hide(): void {
     this.overlayRef.detach();
```

Comment:
Make a separate, private function called `hide()`.

Call this event handler `onMouseLeave()`.